frontend/src/metabase/models/components/ModelDetailPage/ModelActionDetails/ModelActionDetails.tsx

```diff
@@ -8,26 +10,42 @@ import Icon from "metabase/components/Icon";
 import Actions from "metabase/entities/actions";
 import { humanize } from "metabase/lib/formatting";
 
-import type { ModelAction } from "metabase-types/api";
-import type { State } from "metabase-types/store";
-import type Question from "metabase-lib/lib/Question";
+import type { WritebackAction } from "metabase-types/api";
+import type { Dispatch, State } from "metabase-types/store";
 
 import {
   EmptyStateContainer,
   EmptyStateTitle,
 } from "../ModelDetailPage.styled";
 import { ActionListItem, ActionTitle } from "./ModelActionDetails.styled";
+import { hasImplicitActions, isImplicitAction } from "./utils";
+
+const mapDispatchToProps = {
+  enableImplicitActionsForModel: Actions.actions.enableImplicitActionsForModel,
+};
 
 interface Props {
-  model: Question;
-  actions: ModelAction[];
+  modelId: number;
+  actions: WritebackAction[];
+  enableImplicitActionsForModel: (modelId: number) => void;
 }
 
-function ModelActionDetails({ actions }: Props) {
+function ModelActionDetails({
+  actions,
+  modelId,
+  enableImplicitActionsForModel,
+}: Props) {
+  const createImplicitActions = async () => {
+    await enableImplicitActionsForModel(modelId);
+  };
```

Comment:
I guess it's more common here to use names `handleCreateImplicitActions` kind of naming for callbacks inside the component